odoo/addons/base/models/ir_actions_report.py

```diff
@@ -254,16 +248,6 @@ def get_wkhtmltopdf_state(self):
         '''
         return wkhtmltopdf_state
 
-    @api.model
-    def datamatrix_available(self):
```

Comment:
you can't remove the method in a stable version, it's used in report and will only be removed when you update the module. Instead you can keep it and replace the body with `return True`